src/lib/updating-element.ts

```diff
@@ -165,7 +165,7 @@ export const defaultConverter: ComplexAttributeConverter = {
         return value === null ? null : Number(value);
       case Object:
       case Array:
-        return JSON.parse(value!);
+        return JSON.parse(value!) as unknown;
```

Comment:
Can you add a comment here so we don't loose this cast accidentally?

Reply:
Done.